src/test/java/com/gs/crdtools/BUILD

```diff
@@ -18,6 +18,7 @@ java_junit5_test(
         "//src/main/java/com/gs/crdtools:openapi-src-genner",
         "//src/main/java/com/gs/crdtools:spec-extractor",
         "//src/main/java/com/gs/crdtools/codegen:swagger-codegen-extensions",
+        "//src/test/java/com/gs/crdtools/codegen:tests",
```

Comment:
Avoid having a less specific module depend on a more specific one. I would put the Result record definition in a `Results.java` in test/java/com/gs/crdtools, add it to the list of sources in the tests target and depend on that target in the codegen tests

Reply:
I was thinking about this, but I remembered your words about creating utility classes only when there is actually more than one function that we are using, hence why I did not do that. My understanding now is that this rule can be _broken_ when there is a situation like this of a less specific module being dependent on a more specific one. Understood!